src/value/owned.rs

```diff
@@ -514,6 +491,15 @@ impl From<BigInt> for Element {
     }
 }
 
+impl From<Integer> for Element {
+    fn from(integer_val: Integer) -> Self {
+        match integer_val {
+            Integer::I64(i64_int) => i64_int.into(),
+            Integer::BigInt(big_int) => big_int.into(),
+        }
```

Comment:
🗺️ We already had `From<i64>` (e.g. `16.into()`) and `From<BigInt>`, adding `From<Integer>` allows simple conversions in places where we've already constructed an `Integer` enum.